src/xtgeo/grid3d/grid_properties.py

```diff
@@ -178,6 +405,8 @@ def props(self):
     @props.setter
     def props(self, propslist):
         self._props = propslist
+        self.names = [p.name for p in self._props]
```

Comment:
This will trigger deprecation-warnings - not sure if that is intentional. Consider using cache

Reply:
Changed to self._names